go/vt/vttablet/tabletserver/messager/message_manager.go

```diff
@@ -366,37 +366,37 @@ func (mm *messageManager) Open() {
 
 // Close stops the messageManager service.
 func (mm *messageManager) Close() {
-	log.Infof("messageManager - started execution of Close")
+	log.Infof("messageManager (%s) - started execution of Close", mm.name.String())
```

Comment:
Nit, but it might be nicer/safer to use `%v` with `mm.name`. The result should be the same (Stringer interface) but is also nil safe (in case name is somehow nil).

Reply:
Done